packages/react-resizable-panels/src/PanelResizeHandle.test.tsx

```diff
@@ -72,6 +82,26 @@ describe("PanelResizeHandle", () => {
     expect(element.title).toBe("bar");
   });
 
+  it("resets the global cursor style on unmount", () => {
+    act(() => {
+      root.render(
+        <PanelGroup direction="horizontal">
+          <Panel />
+          <PanelResizeHandle />
+          <Panel />
+        </PanelGroup>
+      );
+    });
+
+    expect(cursorUtils.resetGlobalCursorStyle).not.toHaveBeenCalled();
+
+    act(() => {
+      root.unmount();
+    });
+
+    expect(cursorUtils.resetGlobalCursorStyle).toHaveBeenCalled();
+  });
+
```

Comment:
This test should probably be one of the end to end tests. Right now it's not _really_ testing the right behavior (since the cursor is never set in the first place, because there are no intersecting handles in this test)